packages/common/browser/gated-download-wufoo.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+  <div class="button-wrapper" v-if="canDownload">
+    <p>Your download should start automatically. If not, click the button below to access this document.</p>
+    <a :href="target" class="btn btn-lg btn-primary" target="_blank" rel="noopener noreferer">{{ label }}</a>
+  </div>
+  <div v-else>
+    <strong>To access this piece of premium content, please fill out the following form:</strong>
+    <div :id="formId" />
+  </div>
+</template>
+
+<script>
+export default {
+  props: {
+    surveyId: {
+      type: String,
+      required: true,
+    },
+    userName: {
+      type: String,
+      default: 'cygnuscorporate',
+    },
+    label: {
+      type: String,
+      default: 'Download',
+    },
+    target: {
+      type: String,
+      required: true,
+    },
+    height: {
+      type: String,
+      default: '1000',
+    }
+  },
+  data: () => ({ canDownload: false }),
+  mounted() {
+    const init = this.init;
+    if (window.WufooForm) {
+      init();
+    } else {
+      const s = document.createElement('script');
+      s.src = 'https://secure.wufoo.com/scripts/embed/form.js';
+      s.onload = s.onreadystatechange = function() {
+        const rs = this.readyState;
+        if (rs && rs != 'complete' && rs != 'loaded') return;
```

Comment:
`!==` please. You can also destructure: `const { readyState} = this` instead of re-assignment